safe_transaction_service/history/models.py

```diff
@@ -1606,6 +1606,70 @@ class SafeContractManager(models.Manager):
     def get_banned_safes(self) -> QuerySet[ChecksumAddress]:
         return self.filter(banned=True).values_list("address", flat=True)
 
+    def get_count_relevant_txs_for_safe(self, address: ChecksumAddress) -> int:
```

Comment:
What do you think to change this method to an staticmethod due is not using anything from the instance class.

Reply:
That's not [how Django Managers work](https://docs.djangoproject.com/en/4.2/topics/db/managers/)